packages/scripts/src/feature-dependency-graph.ts

```diff
@@ -0,0 +1,179 @@
+import type { Feature } from '@wixc3/engine-core/src';
+
+export interface Link {
+    source: string;
+    target: string;
+}
+
+export const buildFeatureLinks = (
+    entry: Feature,
+    visitedFeatures: { [propName: string]: number },
```

Comment:
use Map<string, number> as a type,
because it's safer in line 15 to use `if(`visitedFeatures.has(dep.is))` also a bit clearer